src/ui/pages/SidePage/components/IncomingRequest/components/SignTransactionRequest.tsx

```diff
@@ -74,29 +66,34 @@ const SignTransactionRequest = ({
           alt={requestData.label}
         />
         <h2 className="sign-name">{requestData.label}</h2>
-        <h3 className="sign-subtitle">{signTransaction?.eventName}</h3>
-        <p className="sign-link">{signTransaction?.ownerUrl}</p>
+        <h3 className="sign-subtitle">{signTransaction?.type}</h3>
       </div>
       <div className="sign-transaction-content">
         <CardDetailsBlock
           className="sign-address"
-          title={`${i18n.t("request.signtransaction.transaction.address")}`}
+          title={`${i18n.t("request.signtransaction.identifier")}`}
         >
-          <IonText className="address">{signTransaction?.data.address}</IonText>
+          <IonText className="address">
+            {signTransaction?.payload.identifier}
+          </IonText>
         </CardDetailsBlock>
         <CardDetailsBlock
           className="sign-data"
           title={i18n.t("request.signtransaction.transaction.data")}
         >
-          <CardDetailsAttributes
-            data={signDetails}
-            itemProps={{
-              mask: false,
-              fullText: true,
-              copyButton: false,
-              className: "sign-info-item",
-            }}
-          />
+          {isSigningObject ? (
+            <CardDetailsAttributes
+              data={signDetails}
+              itemProps={{
+                mask: false,
+                fullText: true,
+                copyButton: false,
+                className: "sign-info-item",
+              }}
+            />
+          ) : (
+            signDetails.toString()
```

Comment:
Can you attach a screenshot to the description of what this looks like when its just a string?

Reply:
![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142211701/19db2863-5388-4fc1-b632-682f9ac0f01d)

Reply:
OK, this needs to match the new designs I sent - the padding is wrong.

Reply:
This still looks wrong from the video in the description